README.md

```diff
@@ -41,16 +41,14 @@ As mentioned above, `JToric` crucially depends on `NConvex`.  A quick way to sat
 Explicitly, you can reproduce the build process by following the following steps:
 
 1. Start `Julia`
-2. Issue the tollowing two lines of code:
+2. Issue the following code:
 
-    `using GAP`
-    `gap_location = GAP.GAPROOT`
+        using GAP; gap_location = GAP.GAPROOT
 
-2. Remember this path gap_location.
+2. Remember this path `gap_location`.
 3. Exit julia.
 4. `cd $(gap_location)/pkg`
-5. `git clone https://github.com/HereAround/NConvex.git`
-6. `git fetch origin martindevel:martindevel`
```

Comment:
Note that "fetch" just downloads code -- my changes here are assuming that you meant to also tell people to `git checkout martindevel` -- is that right?

Reply:
Yes, that is exactly true. I meant to `checkout martindevel`. Thank you for spotting this!